app/ui/browser/reducers/main-reducers.js

```diff
@@ -11,11 +11,12 @@ specific language governing permissions and limitations under the License.
 */
 
 import assert from 'assert';
+import 'babel-polyfill';
```

Comment:
So we'd have to include babel-polyfill in every file that needs a polyfill? That sounds yucky.